### ISSUE TITLE
[Event] multiple classes

### ISSUE THREAD

**Issue by Tyedye** [amtgard/ORK3#56]
when multiple classes are played at one event by one person, it is not possible to enter them all in the event. Currently we are entering the highest number in the event, and adding others in reconcile, however it would be nice if this were changed to keep the records more clear.

**Comment by BenOvermyer** [comment]
Is this still relevant? @esdraelon

**Comment by esdraelon** [comment]
TyeDye will have to common. It's a not uncommon scenario.

**Comment by Tyedye** [comment]
Yes. There are some kingdom events that are actually setting themselves up with each day of the event as a different event instance for this reason.

**Comment by BenOvermyer** [comment]
Alright, we should accommodate that then, so they don't have to do weird things in order to get around the way the system works.
